scenario_tests/logical_replication/docker-compose.yml

```diff
@@ -47,8 +47,7 @@ services:
       REPLICATION_PASS: 'replicator'
       REPLICATION: 'true'
     depends_on:
-        pg-publisher:
-          condition: service_healthy
+        - pg-publisher
```

Comment:
I had to explicitly do this. Because we don't want to wait for `pg-subscriber` to be up to show the logs. If something were happen to the service (causing `pg-publisher` not healthy), it will just wait block unless we declare it like this. So this is necessary for testing purposes.

Reply:
Maybe we could upgrade docker-compose from 2.1 to the latest version. The condition of service has been deprecated